### PR TITLE
Updated dependencies, namings, and wording issues

### PR DESCRIPTION
**Pull request by SandraAhlgrimm**
recreated PR as the original got merged before any reviews

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ languages:
 You will find here a full workshop on Azure Spring Apps, including guides and demos.
 You can run this workshop with the monthly free grant from Azure Spring Apps. [Learn More](https://aka.ms/costs-less)
 
-This lab is based on a public workshop created by [Julien Dubois](https://twitter.com/juliendubois) and available for free to everyone, under the [MIT license](LICENSE.txt).
+This lab is based on a public workshop created by [Julien Dubois](https://twitter.com/juliendubois) and updated by [Mark Heckler](https://twitter.com/mkheck). It is available for free to everyone under the [MIT license](LICENSE.txt).
```
Comment (SandraAhlgrimm) [inline README.md:12]: If you start this, we would have to list all updaters. Let's better link to https://github.com/microsoft/azure-spring-apps-training/graphs/contributors

File: 12-making-microservices-talk-to-each-other/README.md [hunk 2]
```diff
@@ -187,16 +187,16 @@ public class AllCitiesWeatherController {
...
+spring.cloud.openfeign.client.config.default.readTimeout=160000000
 ```
 
 ## Create the application on Azure Spring Apps
 
 As before, create a specific `all-cities-weather-service` application in your Azure Spring Apps instance:
 
 ```bash
-az spring app create -n all-cities-weather-service --runtime-version Java_17
+az spring app create -n all-cities-weather-service --runtime-version Java_17 -s $AZ_SPRING_APPS_NAME -g $AZ_RESOURCE_GROUP
```
Comment (SandraAhlgrimm) [inline 12-making-microservices-talk-to-each-other/README.md:199]: those parameters are not needed to set as we configured those before

File: 08-build-a-spring-cloud-gateway/README.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ A Spring Cloud gateway allows you to selectively expose your microservices and t
 
 The application that we create in this guide is [available here](gateway/).
 
-To create our gateway, we will invoke the Spring Initalizer service from the command line:
+To create our gateway, we will invoke the Spring Initalizr service from the command line:
 
 ```bash
-curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=cloud-gateway,cloud-eureka,cloud-config-client -d baseDir=gateway -d bootVersion=2.7.5 -d javaVersion=17 | tar -xzvf -
+curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=cloud-gateway,cloud-eureka,cloud-config-client -d baseDir=gateway -d bootVersion=3.1.1 -d javaVersion=17 | tar -xzvf -
```
Comment (SandraAhlgrimm) [inline 08-build-a-spring-cloud-gateway/README.md:16]: bootVersion=3.1.3 was used later

File: 07-build-a-spring-boot-microservice-using-mysql/README.md [hunk 4]
```diff
@@ -82,7 +81,7 @@ Now that we've provisioned the Azure Spring Apps instance and configured the ser
 To create our microservice, we will invoke the Spring Initalizer service from the command line:
 
 ```bash
-curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web,data-jpa,mysql,cloud-eureka,cloud-config-client -d baseDir=weather-service -d bootVersion=2.7.5 -d javaVersion=17 | tar -xzvf -
+curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web,data-jpa,mysql,cloud-eureka,cloud-config-client -d baseDir=weather-service -d bootVersion=3.1.1 -d javaVersion=17 | tar -xzvf -
```
Comment (SandraAhlgrimm) [inline 07-build-a-spring-boot-microservice-using-mysql/README.md:84]: bootVersion=3.1.3 is displayed in pom.xml

File: 05-build-a-spring-boot-microservice-using-spring-cloud-features/README.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ The microservice that we create in this guide is [available here](spring-cloud-m
 To create our microservice, we will invoke the Spring Initalizer service from the command line:
 
 ```bash
-curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web,cloud-eureka,cloud-config-client -d baseDir=spring-cloud-microservice -d bootVersion=2.7.5 -d javaVersion=17 | tar -xzvf -
+curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web,cloud-eureka,cloud-config-client -d baseDir=spring-cloud-microservice -d bootVersion=3.1.1 -d javaVersion=17 | tar -xzvf -
```
Comment (SandraAhlgrimm) [inline 05-build-a-spring-boot-microservice-using-spring-cloud-features/README.md:26]: bootVersion=3.1.3 was used as visible in pom.xml

File: 02-build-a-simple-spring-boot-microservice/README.md [hunk 6]
```diff
@@ -130,7 +130,7 @@ If you need to check your code, the final project is available in the ["simple-m
 Here is the final script to build and deploy everything that was done in this guide:
 
 ```
-curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web -d baseDir=simple-microservice -d bootVersion=2.7.5 -d javaVersion=17 | tar -xzvf -
+curl https://start.spring.io/starter.tgz -d type=maven-project -d dependencies=web -d baseDir=simple-microservice -d bootVersion=3.1.1 -d javaVersion=17 | tar -xzvf -
```
Comment (SandraAhlgrimm) [inline 02-build-a-simple-spring-boot-microservice/README.md:133]: bootVersion=3.1.3 was used as visible in pom.xml

**Review by SandraAhlgrimm — COMMENTED** [review]
Thanks @mkheck, let's address the comments below before merging.

**Comment by mkheck** [comment]
Per request by @SandraAhlgrimm, squashing+merging changes previously merged and then extracted for review.

**Pull request merged**